### DISCUSSION TITLE
`Top Pages` statistics Can't load next pages

### DISCUSSION THREAD

**Discussion by kumfo** [plausible/analytics#1737] [Self-Hosted Support]
### Past Issues Searched

- [X] I have searched open and closed issues to make sure that the bug has not yet been reported

### Issue is a Bug Report

- [X] This is a bug report and not a feature request, nor asking for self-hosted support

### Describe the bug
from the error log:
```shell
** (exit) an exception was raised:

** (ArithmeticError) bad argument in arithmetic expression

(plausible 0.0.1) lib/plausible_web/controllers/api/stats_controller.ex:494: PlausibleWeb.Api.StatsController.calculate_cr/2

(plausible 0.0.1) lib/plausible_web/controllers/api/stats_controller.ex:615: anonymous fn/3 in PlausibleWeb.Api.StatsController.add_cr/3

(elixir 1.12.2) lib/enum.ex:1582: Enum."-map/2-lists^map/1-0-"/2

(plausible 0.0.1) lib/plausible_web/controllers/api/stats_controller.ex:344: PlausibleWeb.Api.StatsController.pages/2

(plausible 0.0.1) lib/plausible_web/controllers/api/stats_controller.ex:1: PlausibleWeb.Api.StatsController.action/2

(plausible 0.0.1) lib/plausible_web/controllers/api/stats_controller.ex:1: PlausibleWeb.Api.StatsController.phoenix_controller_pipeline/2

(plausible 0.0.1) lib/plug/error_handler.ex:65: PlausibleWeb.Api.StatsController.call/2

(phoenix 1.5.13) lib/phoenix/router.ex:352: Phoenix.Router.__call__/2
```
I deploy with docker or latest tag.

![ab646dd318cb5d326f154efc52d7a33](https://user-images.githubusercontent.com/7692591/157182857-1fe19cbe-3cda-4c8d-b683-867074347de8.png)

![d4863d445f796181cd56d2177f9b440](https://user-images.githubusercontent.com/7692591/157183032-2e395fef-c3c7-404c-aff1-269daddd94c0.png)
![3ee7cd66efd5d396159126ef2684a49](https://user-images.githubusercontent.com/7692591/157183068-c6eeeec2-2c33-40a4-a179-1de4ee018e81.png)

response error:

```json
{"error":"** (ArithmeticError) bad argument in arithmetic expression"}
```

### Expected behavior

I want to load next page statistics.

### Environment

```markdown
- OS: windows
- Browser: Chrome
- Browser Version:98.0.4758.102
```

**Comment by adamekberg** [comment]
I am getting the same error. Top pages stuck with loading symbol when filtering on a goal. Any news here?
